src/Apps/Order2/Utils/__tests__/addressUtils.jest.ts

```diff
@@ -94,59 +94,31 @@ describe("addressUtils", () => {
 
   describe("formatPhoneNumber", () => {
     it("formats phone number with country code", () => {
-      const address = {
-        phoneNumber: "1234567890",
-        phoneNumberCountryCode: "US",
-      }
-      expect(formatPhoneNumber(address)).toBe("+1 1234567890")
+      expect(formatPhoneNumber("1234567890", "US")).toBe("+1 1234567890")
     })
 
     it("handles different country codes", () => {
-      const address = {
-        phoneNumber: "1234567890",
-        phoneNumberCountryCode: "GB",
-      }
-      expect(formatPhoneNumber(address)).toBe("+44 1234567890")
+      expect(formatPhoneNumber("1234567890", "GB")).toBe("+44 1234567890")
     })
 
     it("returns empty string when phone number is missing", () => {
-      const address = {
-        phoneNumber: null,
-        phoneNumberCountryCode: "US",
-      }
-      expect(formatPhoneNumber(address)).toBe("")
+      expect(formatPhoneNumber(null, "US")).toBe("")
     })
 
-    it("returns empty string when country code is missing", () => {
-      const address = {
-        phoneNumber: "1234567890",
-        phoneNumberCountryCode: null,
-      }
-      expect(formatPhoneNumber(address)).toBe("")
+    it("returns phone number when country code is missing", () => {
```

Comment:
If I understand correctly, this is the behavioral change of this PR. Is EMI-2663 the correct ticket this PR solves?